snapcraft/models/project.py

```diff
@@ -946,44 +1678,99 @@ class Project(models.Project):
 
         Required if one of ``version``, ``summary``, or ``description`` isn't set."""
         ),
-        examples=["adopt-info: foo-part"],
+        examples=["foo-part"],
     )
+    """Selects a part to inherit metadata from and reuse for the snap's
+    metadata.
+
+    Required if one of ``version``, ``summary``, or ``description`` isn't set.
+
+    There are two ways to adopt metadata from a part:
+
+    - Using the 'craftctl set' command in an 'override-' script.
+    - Using appstream metadata files listed in the 'parse-info'  field of the
+      adopting part.
+    """
+
     system_usernames: dict[str, Any] | None = pydantic.Field(
         default=None,
-        description="The system usernames that the snap can use to run services.",
-        examples=["system-usernames: {snap-daemon: shared}"],
+        description="The system usernames that the snap can use to run daemons and services.",
+        examples=["{snap-daemon: shared}"],
```

Comment:
Is there a reason some examples show up as one line?  This one shows

```yaml
system-usernames: snap-daemon: shared
```

Not that it's wrong.  I'm using to seeing multiple lines in project files and am curious if this is by design or unintentional.